### ISSUE TITLE
Common.Logging.NLog not available for net core

### ISSUE THREAD

**Issue by jwaimann** [net-commons/common-logging#147]
I'm currently using Common.Logging version 3.4.0-beta2 and NLog 5.0.0-beta07 on netcoreapp1.1.
I cannot find Common.Logging.NLog specific version for that case.

In case it is not yet available, which log file option do I have as adapter for net core?

Thanks in advance.

**Comment by LHCGreg** [comment]
@jwaimann : What I did is create an NLog adapter compatible with netstandard. See https://github.com/LHCGreg/animerecs/tree/589ff87/Common.Logging.NLogNetStandard. I just had to take the files that Common.Logging.NLog444 uses, make two modifications, and reference NLog 5.0.0-beta09 and Common.Logging 3.4.0-Beta2.

The two modifications were:

* Remove NLogLoggerFactoryAdapter constructor that takes a System.Collections.Specialized.NameValueCollection, as that type is not present in netstandard. I didn't bother with `#if`'ing that constructor since the code is just for me.

* In NLogLoggerFactoryAdapter constructor, use AppContext.BaseDirectory instead of AppDomain.CurrentDomain.BaseDirectory because AppDomain does not exist in netstandard. If targeting the full .NET framework, AppContext requires .NET 4.6, so this also would have to be `#if`'ed to make it suitable for an official package.

I'd submit a PR but I'm not sure how the maintainers want to do multitargeting. Using the new csproj project style ("SDK style") would allow much easier multitargeting than the current setup with .DotNetCore, .Silverlight, etc projects.
